apps/src/craft/craft.js

```diff
@@ -716,11 +716,16 @@ Craft.reportResult = function (success) {
 
   var keepPlayingText = Craft.replayTextForResult(testResultType);
 
+  const image = Craft.initialConfig.level.freePlay ?
+      Craft.gameController.getScreenshot() : null;
+  const encodedImage = image ? encodeURIComponent(image.split(',')[1]) : null;
```

Comment:
It strips out the metadata at the front of a data URI:

![image](https://cloud.githubusercontent.com/assets/206973/20075817/d0c639e4-a503-11e6-9dd8-a3bfb8737675.png)

(This gets done in all the instances we convert screenshots into URI-encoded data to send along—)

![image](https://cloud.githubusercontent.com/assets/206973/20075674/38f1ce8a-a503-11e6-827f-a365fece7184.png)